server/src/modules/api/ombi-api/helpers/ombi-api.helper.ts

```diff
@@ -0,0 +1,104 @@
+import { Injectable } from '@nestjs/common';
+import axios, { AxiosInstance } from 'axios';
+import { SettingsService } from '../../../settings/settings.service';
+import { MaintainerrLogger } from '../../../logging/logs.service';
+import { BasicResponseDto } from '../../external-api/dto/basic-response.dto';
+
+@Injectable()
+export class OmbiApi {
+  constructor(
+    private readonly settingsService: SettingsService,
+    private readonly logger: MaintainerrLogger,
+  ) {
+    this.logger.setContext(OmbiApi.name);
+  }
+
+  private async getApiClient(): Promise<AxiosInstance> {
+    const settings = await this.settingsService.getSettings();
+    
+    if (settings instanceof BasicResponseDto || !settings.ombi_url || !settings.ombi_api_key) {
+      throw new Error('Ombi is not configured');
+    }
+
+    return axios.create({
+      baseURL: `${settings.ombi_url}/api/v1`,
+      headers: {
+        'Content-Type': 'application/json',
+        ApiKey: settings.ombi_api_key,
+      },
+      timeout: 10000,
+    });
+  }
+
+  async testConnection(): Promise<BasicResponseDto> {
+    try {
+      const apiClient = await this.getApiClient();
+      const response = await apiClient.get('/Status');
```

Comment:
Use `/api/v1/settings/about` instead. This returns the version in the body, as well as requiring authentication so is a better test here. A 403 is returned when the API key is incorrect, see `server\src\modules\api\overseerr-api\overseerr-api.service.ts` `validateApiConnectivity` on how this is returned to the user.